notifications_utils/formatters.py

```diff
@@ -338,21 +338,35 @@ def insert_action_link(markdown: str) -> str:
     return re.sub(r'''(>|&gt;){2}\[([\w -]+)\]\((\S+)\)''', substitution, markdown)
 
 
-def insert_block_quotes(md: str) -> str:
-    """
-    Template markup uses ^ to denote a block quote, but Github markdown, which Mistune reflects, specifies a block
-    quote with the > character.  Rather than write a custom parser, templates should preprocess their text to replace
-    the former with the latter.  This preprocessing should take place before any manipulation by Mistune.
+def insert_action_link_block_quote(markdown: str) -> str:
+    img_src = get_action_link_image_url()
 
-    Given:
-        ^ This is a block quote.
+    def replacement(match):
+        """Dynamically constructs the replacement string."""
+        link_html = (
+            f'<a href="{match.group(3)}">'
+            f'<img alt="call to action img" src="{img_src}" style="{ACTION_LINK_IMAGE_STYLE}"> '
+            f'<b>{match.group(2)}</b></a>'
+        )
 
-    Output:
-        > This is a block quote.
-    """
+        extra_text = match.group(4).strip() if match.group(4) else ""
+        if extra_text:
+            link_html += f'<br />{extra_text}'
+
+        return link_html
+
+    return re.sub(r'''(>|&gt;){2}\[([\w -]+)\]\((\S+)\)(.*)?''', replacement, markdown)
+
+
+def insert_block_quotes(md: str) -> str:
+    modified_md = md
+    block_quote_matches = list(re.finditer(r'^(?:\^|>)(?!>).*', modified_md, flags=re.MULTILINE))
 
-    return re.sub(r'''^(\s*)\^(\s*)''', r'''\1>\2''', md, flags=re.M)
+    for match in block_quote_matches:
```

Comment:
Note that this iterates over the original input; not the modified copy.
```suggestion
    for match in re.finditer(r'^(?:\^|>)(?!>).*', md, flags=re.MULTILINE):
```